tensorflow/lite/core/macros.h

```diff
@@ -65,4 +65,14 @@ limitations under the License.
 #define TFLITE_HAS_ATTRIBUTE_WEAK 0
 #endif
 
+#ifndef TF_LITE_STATIC_MEMORY
```

Comment:
Do we use this flag to gate any differences between TFLite & TFLM? The naming is kind of non-intuitive in this context, as its guarding static data.

Reply:
it is indeed confusing. http://b/145762662 and http://b/160955687 has some historical context.